pkg/utils/utils_test.go

```diff
@@ -224,3 +225,278 @@ func TestIsEqual(t *testing.T) {
 		t.Errorf("Expected %v, get %v", expectedErr, err.Error())
 	}
 }
+
+type animal struct {
```

Comment:
Animal ;)

Reply:
it's for unit testing only, so I use lowercase here